idsvr/templates/post-hook.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.postHook.enabled -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "curity.fullname" . }}-post-hook
+  labels:
+    chart: {{ include "curity.chart" . }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook-weight": "10"
+spec:
+  backoffLimit: 1
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: {{ include "curity.fullname" . }}-post-hook
+          image: {{ .Values.postHook.image }}
+        {{- if hasKey .Values.postHook "command" }}
+          command:
+          {{- range .Values.postHook.command }}
```

Comment:
Hmm. I think I was thinking that anyone wanting to change the `command` will have looked into the chart anyway... But since you are asking I understand that it is not that clear. I will add it.